features/admin/acknowledging_contributors_site.feature

```diff
@@ -9,7 +9,7 @@ Feature: Acknowledging Contributors
     Given I visit the <page> page
     Then I should see the "Agile Ventures" image linked to contributors
 #    And I should see the "Do-it" image linked to "https://do-it.org/"
-    And I should see the "Ssl-secure-badge" image linked to "https://www.expeditedssl.com/simple-ssl-scanner/scan?target_domain=www.harrowcn.org.uk"
+    scan?target_domain=www.harrowcn.org.uk"
```

Comment:
Seems you forgot to remove this line `scan?target_domain=www.harrowcn.org.uk"`

Reply:
Yeah true. Noted